pkg/console/jsre/completion.go

```diff
@@ -0,0 +1,66 @@
+package jsre
```

Comment:
retain the go-ethereum license: https://github.com/ethereum/go-ethereum/blob/master/internal/jsre/completion.go#L1